assets/js/hocs/with-feedback-prompt/index.js

```diff
@@ -11,41 +10,25 @@ import { createHigherOrderComponent } from '@wordpress/compose';
  */
 import FeedbackPrompt from './feedback-prompt.js';
 
-const blocksFeedback = {
-	'woocommerce/cart': __(
-		'We are currently working on improving our checkout and providing merchants with tools and options to customize their checkout to their stores needs.',
-		'woo-gutenberg-products-block'
-	),
-	'woocommerce/checkout': __(
-		'We are currently working on improving our checkout and providing merchants with tools and options to customize their checkout to their stores needs.',
-		'woo-gutenberg-products-block'
-	),
-};
-
 /**
  * Adds a feedback prompt to the editor sidebar.
  *
  * @param {WPComponent} BlockEdit Original component.
  *
  * @return {WPComponent} Wrapped component.
  */
-const withFeedbackPrompt = createHigherOrderComponent( ( BlockEdit ) => {
-	return ( props ) => {
-		const feedbackPromptText = blocksFeedback[ props.name ];
-
-		if ( feedbackPromptText && props.isSelected ) {
-			return (
-				<Fragment>
-					<BlockEdit { ...props } />
-					<InspectorControls>
-						<FeedbackPrompt text={ feedbackPromptText } />
-					</InspectorControls>
-				</Fragment>
-			);
-		}
-
-		return <BlockEdit { ...props } />;
-	};
-}, 'withFeedbackPrompt' );
+const withFeedbackPrompt = createHigherOrderComponent(
+	( content ) => ( BlockEdit ) => {
```

Comment:
I think this should be:

```js
const withFeedbackPrompt = ( content ) => createHigherOrderComponent( /**/ );
```

In other words, the curried function should return the higher order component.

Reply:
Yes... I wasn't sure about this. Changed in: d016e8ff997d7bf23a961e2f4d3addec24c07a93.